### PR TITLE
Add documentation for Merchants & Templates clients. CH-17154

### PR DESCRIPTION
**Pull request by davehensleyns8**

### CODE REVIEW

File: public/en/platform/protect-sdk-php/merchants-documentation.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+## Merchants Documentation
```
Comment (crfroehlich) [inline public/en/platform/protect-sdk-php/merchants-documentation.md]: Markdown linter will complain that this starts with `##` and not `#`

File: public/en/platform/protect-sdk-php/merchants-documentation.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+### Purpose of the Merchants Client
+The purpose of the Merchants class is to easily acquire information about the current merchant. This is necessary in order to load the Protect client iframe.
```
Comment (crfroehlich) [inline public/en/platform/protect-sdk-php/merchants-documentation.md]: Linter will complain that there isn't a new line after the `###` block

**Comment by davehensleyns8** [comment]
> Two minor lint issues; otherwise looks good.

Thanks @crfroehlich , I think all of the existing docs have these same issues since they are all formatted the same way. Should I make a ticket to just go through and fix em all in one shot?

**Comment by crfroehlich** [comment]
@davehensleyns8 yeah, just make a ticket to fix them all in one pass.

**Comment by davehensleyns8** [comment]
> @davehensleyns8 yeah, just make a ticket to fix them all in one pass.

https://app.clubhouse.io/ns8/story/19924/fix-markdown-formatting-in-php-sdk-documentation

**Pull request merged**